modules/imgproc/src/color_yuv.simd.hpp

```diff
@@ -1825,18 +1828,19 @@ void cvtTwoPlaneYUVtoBGR(const uchar * src_data, size_t src_step,
     CV_INSTRUMENT_REGION();
```

Comment:
Looks like either this function is redundant or HAL's function

> CV_EXPORTS void cvtTwoPlaneYUVtoBGR(const uchar * src_data, size_t src_step,
>                                     uchar * dst_data, size_t dst_step,
>                                     uchar * dst_data, size_t dst_step,
>                                     int dst_width, int dst_height,
>                                     int dst_width, int dst_height,
>                                     int dcn, bool swapBlue, int uIdx);
>                                     int dcn, bool swapBlue, int uIdx);

is redundant.

Reply:
This redundancy is removed in the 2a6e4fd commit.